### PR TITLE
Gather 2023 existing data for 2024 modeling

### PR DESCRIPTION
**Pull request by wrridgeway**

### CODE REVIEW

**Comment by wrridgeway** [comment]
@dfsnow only thing i have left on the update inventory is land site rates. otherwise this is good to go.

File: aws-s3/scripts-ccao-data-raw-us-east-1/spatial-transit.R [hunk 1]
```diff
@@ -78,14 +80,26 @@ walk(metra_feed_dates_list, get_metra_feed)
 ##### Pace #####
 pace_feed_dates_list <- c(
   "2015-10-16", "2016-10-15", "2017-10-16", "2018-10-17",
-  "2019-10-22", "2020-09-23", "2021-03-15"
+  "2019-10-22", "2020-09-23", "2021-03-15", "2023-09-24"
 )
 
 get_pace_feed <- function(feed_date) {
```
Comment (dfsnow) [inline aws-s3/scripts-ccao-data-raw-us-east-1/spatial-transit.R:86]: I think the TransitFeeds DB is deprecated/offline now. Are we still getting all the data we need for this?

**Review by dfsnow — APPROVED** [review]
This looks fine to me @wrridgeway. I assume you've gone through the outputs to make sure they're up-to-snuff.

File: aws-s3/scripts-ccao-data-raw-us-east-1/spatial-transit.R (hunk 1 above)
Comment (wrridgeway) [reply aws-s3/scripts-ccao-data-raw-us-east-1/spatial-transit.R:86]: Yep, these late 2023 dates and their data are present in the Athena tables.

**Pull request merged**